README.md

```diff
@@ -1,7 +1,10 @@
-#TAL -- TV Application Layer
-[<img src="https://api.travis-ci.org/fmtvp/tal.png?branch=master" align="right" alt="Build Status" />](https://travis-ci.org/fmtvp/tal)
```

Comment:
I removed this `img` tag for the travis build status and replaced it with the Markdown version that travis provide as it seemed to cause layout issues.